### PR TITLE
First add ticket @ADVSYS-10 Please review. Thank you.

### PR DESCRIPTION
**Pull request by PailinC**

### CODE REVIEW

File: src/test/resources/features/home.feature [hunk 1]
```diff
@@ -112,6 +112,28 @@ Feature:Home page related scenarios
...
+    Scenario Outline: Verify social media buttons are displayed
+      Then Verify <LinkBtns> Btns are displayed
+      Examples:
+        | LinkBtns |
+        | facebook        |
+        | twitter         |
+        | google-plus     |
+        | linkedin        |
+
+  @ADVSYS-10
```
Comment (MaksymBuzin) [inline src/test/resources/features/home.feature:125]: That is not good. You just needed verify titles. Like When user clicks on social link <link>. Then Verify title of the page is <title>. To verify title we already have a step so just use it please.

File: src/test/java/step_definitions/HomeSteps.java [hunk 2]
```diff
@@ -162,5 +162,66 @@ public void verifyIsPresent(String header) {
...
+    public void verify_linkedin_btns_are_displayed() {
+        Assert.assertTrue(WebDriverManager.isDisplayed(homePage.linkedinBtn));
+    }
+    @Then("Verify Btns can connect to the same https:\\/\\/www.facebook.com\\/")
+    public void verify_btns_can_connect_to_the_same_https_www_facebook_com() {
+       Assert.assertEquals("https://www.facebook.com/", WebDriverManager.getDriver().getCurrentUrl());
+    }
+    @When("facebook Btns are clickable")
+    public void facebook_btns_are_clickable() {
+        WebDriverManager.click(homePage.facebookBtn);
```
Comment (MaksymBuzin) [inline src/test/java/step_definitions/HomeSteps.java:195]: Step @When("facebook Btns are clickable") and you just perform a click ? that is weird

File: src/test/java/step_definitions/HomeSteps.java [hunk 3]
```diff
@@ -162,5 +162,66 @@ public void verifyIsPresent(String header) {
...
+    }
+
+
+    @Then("Verify linkedin Btns are displayed")
+    public void verify_linkedin_btns_are_displayed() {
+        Assert.assertTrue(WebDriverManager.isDisplayed(homePage.linkedinBtn));
+    }
+    @Then("Verify Btns can connect to the same https:\\/\\/www.facebook.com\\/")
+    public void verify_btns_can_connect_to_the_same_https_www_facebook_com() {
+       Assert.assertEquals("https://www.facebook.com/", WebDriverManager.getDriver().getCurrentUrl());
```
Comment (MaksymBuzin) [inline src/test/java/step_definitions/HomeSteps.java:191]: delete it. Wrong step.

File: src/test/java/step_definitions/HomeSteps.java [hunk 4]
```diff
@@ -162,5 +162,66 @@ public void verifyIsPresent(String header) {
         }
 
     }
+
```
Comment (MaksymBuzin) [inline src/test/java/step_definitions/HomeSteps.java:165]: you can put all these in switch

File: src/test/resources/features/home.feature (hunk 1 above)
Comment (PailinC) [reply src/test/resources/features/home.feature:125]: Oh I see, Will try to revise and get it done late. Thanks for the review.

File: src/test/resources/features/home.feature [hunk 5]
```diff
@@ -112,6 +112,28 @@ Feature:Home page related scenarios
...
+  Scenario Outline: Verify Each buttons should take the user to corresponding page
+    When <LinkBtns> Btns are clickable
```
Comment (MaksymBuzin) [inline src/test/resources/features/home.feature:127]: you declared <LinkBtns> but you missed quotes  . The statement doesn't do anything.

**Pull request closed without merging**